src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
 <lib-header></lib-header>
 <router-outlet />
+<!---->
```

Comment:
Algum motivo para este comentário?

Reply:
Fala Charles, fizemos esse comentário para subir o Merge da Main com a Branch que estava desatualizada. Vou pedir para ele remover...
Fiz essa ISSUE junto com ele, pra da uma força.

Reply:
@charlesribeiro removido comentário conforme observação colocada.